Modules/CustomerHistory/Frames.lua

```diff
@@ -58,15 +89,9 @@ function CraftSim.CUSTOMER_HISTORY.FRAMES:Init()
     createContent(self.frame)
 end
 
-function CraftSim.CUSTOMER_HISTORY.FRAMES:AddCustomer(customer)
-    self:ResetDropdown(customer)
-end
-
 local function GetFallbackCustomer()
-    for k, v in pairs(CraftSim.CUSTOMER_HISTORY.db.realm) do
-        if type(v) == "table" then
-            return k
-        end
+    if CraftSim.CUSTOMER_HISTORY.db.realm.lastCustomer then
+        return CraftSim.CUSTOMER_HISTORY.db.realm.lastCustomer
```

Comment:
Again I see no point in keeping the `GetFallbackCustomer` method with this change.